example-scripts/bash-scripts/full-harvest-examples/1.13-1.15-examples/example-openalex/openalex-to-vivo.datamap.xsl

```diff
@@ -0,0 +1,1513 @@
+                <bibo:pageStart><xsl:value-of select="$pageStart" /></bibo:pageStart>
+            </xsl:if>
+            <xsl:if test="$pageEnd != ''">
+                <bibo:pageEnd><xsl:value-of select="$pageEnd" /></bibo:pageEnd>
+            </xsl:if>
+            <xsl:if test="$is_oa != ''">
+                <xsl:choose>
+                    <xsl:when test="$is_oa='true'">
+                        <vivo-oa:has_access rdf:resource="http://lod.tib.eu/onto/vivo-oa/Open_Access"/>
+                    </xsl:when>
+                    <xsl:when test="$is_oa='false'">
+                        <vivo-oa:has_access rdf:resource="http://lod.tib.eu/onto/vivo-oa/Non_Open_Access"/>
+                    </xsl:when>
+                </xsl:choose>
+            </xsl:if>
+            <xsl:if test="$oa_status != ''">
+                <xsl:choose>
+                    <xsl:when test="$oa_status='green'">
+                        <vivo-oa:has_access rdf:resource="http://lod.tib.eu/onto/vivo-oa/Open_Access_Green"/>
+                    </xsl:when>
+                    <xsl:when test="$oa_status='gold'">
+                        <vivo-oa:has_access rdf:resource="http://lod.tib.eu/onto/vivo-oa/Open_Access_Gold"/>
+                    </xsl:when>
+                </xsl:choose>
+            </xsl:if>
+            <xsl:if test="$cited_by_count != ''">
+                <c4o:hasGlobalCitationFrequency rdf:resource="{$baseURI}gcf_{$oaid}"/>
+            </xsl:if>
+
+            <xsl:for-each select="$this/node-publication:authorships/*">
+                <xsl:variable name="id" select="substring-after(author/id,'org/')"/>
+                <core:relatedBy rdf:resource="{$baseURI}authorship_{$oaid}"/>
+            </xsl:for-each>
+
+            <xsl:for-each select="$this/node-publication:concepts/*">
+                <xsl:call-template name='getConcept'>
+                    <xsl:with-param name = "concept" select = "id" />
+                </xsl:call-template>
+            </xsl:for-each>
+
+        </rdf:Description>
+
+        <!-- date time value -->
+        <xsl:if test="normalize-space( $publication_date )">
+            <rdf:Description rdf:about="{$baseURI}dtv_{$oaid}">
+                <rdf:type rdf:resource="http://vivoweb.org/ontology/core#DateTimeValue"/>
+                <core:dateTime rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime"><xsl:value-of select="$publication_date"/>T00:00:00</core:dateTime>
+                <core:dateTimePrecision rdf:resource="http://vivoweb.org/ontology/core#yearMonthDayPrecision"/>
+            </rdf:Description>
+        </xsl:if>
+
+        <!-- global citation count -->
+        <xsl:if test="normalize-space( $cited_by_count )">
+            <rdf:Description rdf:about="{$baseURI}gcf_{$oaid}">
+                <rdf:type rdf:resource="http://purl.org/spar/c4o/GlobalCitationCount"/>
+                <c4o:hasGlobalCountSource rdf:resource="https://forschungsatlas01.develop.service.tib.eu/individual/n4885"/>
```

Comment:
TIB specific

Reply:
Have to find out why there is a hardcoded individual URI.